tool/docker/runtime/cpu/Dockerfile

```diff
@@ -28,10 +28,12 @@ RUN apt-get update \
     && rm -rf /var/lib/apt/lists/*
 
 # install conda
-RUN wget --no-check-certificate https://repo.continuum.io/miniconda/Miniconda3-4.5.12-Linux-x86_64.sh -O miniconda.sh;
+RUN wget --no-check-certificate https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
```

Comment:
is it better to fix the version of miniconda?
if miniconda has a new version, is it possible that singa will fail to build/install?

Reply:
yes. thanks. this is updated.